common/hdr-model.cpp

```diff
@@ -0,0 +1,345 @@
+﻿// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2025 Intel Corporation. All Rights Reserved.
+
+#include <glad/glad.h>
+#include "hdr-model.h"
+#include "device-model.h"
+#include "ux-window.h"
+#include <rsutils/os/special-folder.h>
+#include <rsutils/string/hexdump.h>
+#include <imgui.h>
+#include <realsense_imgui.h>
+#include <fstream>
+#include <stdexcept>
+#include <os.h>
+using rsutils::json;
+
+namespace rs2 {
+hdr_config::control_item::control_item()
+    : DepthGain( 1 )
```

Comment:
Best practice - this constructor should call the second constructor with parameter values. Avoids code duplication if changing constructor implementation.